source/api_documentation/network_integration/advertiser_campaigns/index.rst

```diff
@@ -294,12 +292,43 @@ Node Parameters and Usage
 
   * - VerifyLocation
     - prompt
-    - Prompt the caller to verify the guessed location or confirm through input. Useful if geographical data is important or useful in a condition node.
+    - Prompts the caller to verify the guessed location or confirm through input. Useful if geographical data is important or useful in a condition node.
 
   * - DynamicRoute (beta - read only)
     - \*dynamic_route_destination
     - Forwards the call to a destination that is extracted from a custom data field specified in dynamic_route_destination. The destination must be a phone number or if you are SIP integrated, can be a string that is routable by your SIP infrastructure.
 
+  * - AnyKeyPress
+    - \*prompt
+
+    - Allows the caller to make any keypress to continue the call.
+
+  * - NumberQuestion
+    - \*prompt
+
+      \*number_question_type
+
+      confirm_response_enabled
```

Comment:
We'll also need to update the `Menu` params to indicate it supports `confirm_response_enabled`

Reply:
`Menu` also needs a note about what happens when speech recognition is enabled.

Reply:
>  `Menu` also needs a note about what happens when speech recognition is enabled.

I did my best, but would appreciate your review on this in particular.